server/.nvmrc

```diff
@@ -0,0 +1 @@
+16.14.2
```

Comment:
why do we need nvmrc here?

Reply:
I am not sure I understand the question.

Do you want it moved to the root?